src/index.js

```diff
@@ -30,11 +30,8 @@ const ProjectDirectoryUtils = require('./utils/project-directory');
 const { is2FASaltValid } = require('./utils/token-checker');
 const { getJWTConfiguration } = require('./config/jwt');
 
-const pathProjectAbsolute = new ProjectDirectoryUtils().getAbsolutePath();
```

Comment:
In my opinion, the only function that should change is `getAbsolutePath`, as it is its ownership to return the right path according to the context of the project.

The code below can remain the same if `getAbsolutePath` return the right path in the Yarn 2 context.